about.html

```diff
@@ -0,0 +1,112 @@
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="banner">
+        <div class="banner-page">
+            <i class="menu-button"></i>
+            <div class="about-banner-content">
+                <div>
+                    <h1 class="align-center">Welcome to the Sports League<br> Conferance 2023</h1>
+                </div>
+                <p class="brief-description-banner">
+                    Sports league is a unique initiative to promote all sports in region
+                    of India. We
+                    hope you'll consider joining us this year to build a stronger, more
+                    vibrant commons together.
+                </p>
+                <div class="contact-us">
+                    <p>Please contact for further inquiries
+                        about this conference</p>
+                    <span><strong>spccontact2023@sgcindia.org</strong></span>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="about">
+        <h2 class="section-headline">Soprts League Conferance 2023 Logo</h2>
+        <div class="about-us">
+            <p>The logo of Soprts League Conferance was decided through the logo
+                competition from 8,January to 7,April.</p>
+            <img class="about-logo" src="Images/logo.png" />
+        </div>
+        <hr>
+        <h2 class="section-headline">See the past Sports League Conferances</h2>
+        <div class="about-us">
+            <p>Take a look at the past two Sports League Conferance which took place in
+                Indore and in Mumbai</p>
+            <div class="card-wrapper">
+                <div class="previous-summit summit1">
+                    <span>2019</span>
+                    <span>Indore Sports League Conference,
+                        Indore</span>
+                </div>
+                <div class="previous-summit summit2">
+                    <span>2021</span>
+                    <span>Mumbai Sports League Conference,
+                        Mumbai</span>
+                </div>
+            </div>
+        </div>
+
+    </section>
+    <section class="partner-about" id="partner">
+        <h2 class="section-headline">Partner</h2>
+        <ul>
+            <li class="mozilla"></li>
+            <li class="google"></li>
+            <li class="naver"></li>
+            <li class="daumkakao"></li>
+            <li class="airbnb"></li>
+        </ul>
+    </section>
+    <footer>
+        <img src="Images/logo.png" class="logo-footer" />
+        <p>Sports League Conferance <br />Some Rights Reserved</p>
+    </footer>
```

Comment:
- [ ] Good job ensuring that your About page footer and the home page are the same 👍🏽 . I think what's left is for you to ensure that your About page footer has a dark background instead of the white one you used.

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/233504093-e04bcb2b-f3e6-41b3-9572-258e4017031e.png)

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/233504206-466d84a1-0a28-48b0-8e43-a9fb327a389e.png)